scripts/demo/sv4d_helpers.py

```diff
@@ -158,46 +154,120 @@ def read_video(
 
     if len(images) < n_frames:
         images = (images + images[::-1])[:n_frames]
-        
     if len(images) != n_frames:
         raise ValueError(f"Input video contains fewer than {n_frames} frames.")
 
-    # Remove background and crop video frames
     images_v0 = []
-    for t, image in enumerate(images):
+
+    for image in images:
+        image = ToTensor()(image).unsqueeze(0).to(device)
+        images_v0.append(image * 2.0 - 1.0)
+    return images_v0
+
+
+def preprocess_video(input_path, remove_bg=False, n_frames=21, W=576, H=576, output_folder=None, image_frame_ratio = 0.917):
```

Comment:
- Why separate into two functions?
- Where did we get the image_frame_ratio = 0.917 from?

Reply:
- Why separate into two functions?  
That's for gradio demo. To vis the processed image, we show the rgb video instead of normalized tensor. When running the sv3d/sv4d sampling, we convert it to tensor after that.

- Where did we get the image_frame_ratio = 0.917 from?
Manually adjusted image_frame_ratio and 0.917 works best for current camera radius.